INSTALL.md

```diff
@@ -0,0 +1,69 @@
+# Installation
+
+## Current development version
+Custom repositories and requirements must be set at root level during the current development phase.
+Edit the file, and add the following blocks:
+
+```json
+    "require": {
+        "composer/installers": "dev-ezplatform_assets as 1.3.x-dev",
+        "dpobel/ez-field-edit": "dev-master as 1.0.x-dev",
```

Comment:
this is a bit outdated, only composer/installers (and the corresponding repository) is now needed. But maybe, we should tell people to use the branch of https://github.com/ezsystems/ezplatform/pull/177 ?